dbNSFP.pm

```diff
@@ -262,8 +262,10 @@ sub run {
   my $pos;
 
   my $assembly = $self->{config}->{assembly};
- 
-  foreach my $tmp_data(@{$self->get_data($vf->{chr}, $vf->{start} - 1, $vf->{end})}) {
+  
+  my $chr = $vf->{chr};
+  $chr = 'M' if ($chr =~ /MT/i);
```

Comment:
A suggestion for a more condensed code, but your code looks good: 
` my $chr = ($vf->{chr} =~ /MT/i) ? 'M' : $vf->{chr};`